01-shapefile-attributes.Rmd

```diff
@@ -79,12 +80,12 @@ available for 30 years from the NEON flux tower [from the NEON data portal](http
  
 ##Notes about R Libraries
 
-To work with vector data in R, we can use the `rgdal` library. We will load the `raster`
-library to work with rasters. The `raster` library also allows us to explore metadata
-using similar commands with both rasters and vectors.
- 
-##Querying shapefile attributes
+To work with vector data in R, we can use the `rgdal` library. We will load the 
+`raster` library to work with rasters. The `raster` library also allows us to 
+explore metadata using similar commands with both rasters and vectors.
 
+##Setting up the workspace
+If you completed Lesson 00 you can skip this code chunk.
```

Comment:
ADD LINK TO LESSON 1 - note that i'm using site_baseurl which allows the lesson to move to a different repo and the URL will still work!

[ {{site.baseurl }} R/open-shapefiles-in-R/]( Lesson 00 - Lesson name)